sqlx/lexer.ts

```diff
@@ -4,7 +4,9 @@ const LEXER_STATE_NAMES = {
   SQL: "sql",
   JS_BLOCK: "jsBlock",
   JS_TEMPLATE_STRING: "jsTemplateString",
-  INNER_SQL_BLOCK: "innerSqlBlock"
+  INNER_SQL_BLOCK: "innerSqlBlock",
+  INNER_SINGLE_QUOTE: "innerSingleQuote",
```

Comment:
can we call these two `SQL_SINGLE_QUOTE_STRING` and `SQL_DOUBLE_QUOTE_STRING`?

Reply:
Sure